src/javascripts/components/search.mjs

```diff
@@ -99,31 +99,44 @@ Search.prototype.inputValueTemplate = function (result) {
 }
 
 Search.prototype.resultTemplate = function (result) {
-  // add rest of the data here to build the item
+  function startsWithFilter (words, query) {
+    return words.filter(function (word) {
+      return word.trim().toLowerCase().indexOf(query.toLowerCase()) === 0
+    })
+  }
+
+  // Add rest of the data here to build the item
   if (result) {
     var elem = document.createElement('span')
-    var resultTitle = result.title
-    elem.textContent = resultTitle
-    if (result.aliases) {
-      var aliases = result.aliases.split(',').map(function (item) {
-        return item.trim()
-      })
-      var matchedAliases = []
-      // only show a matching alias if there are no matches in the title
-      if (resultTitle.toLowerCase().indexOf(searchQuery) === -1) {
-        // it would be confusing to show the user
-        // aliases that don't match the typed query
-        matchedAliases = aliases.filter(function (alias) {
-          return alias.toLowerCase().indexOf(searchQuery.toLowerCase()) !== -1
```

Comment:
Without a title match, we'd then show too many aliases that "contains" not "begins with" `searchQuery`